scripts/reformat_data_census.py

```diff
@@ -103,6 +103,9 @@ def main():
 
     df_crop.to_csv(FILENAME_REFORMATED, index=False)
 
+    print('copy ' + FILENAME_REFORMATED + ' to /usr/share/ with:')
+    print("sudo cp " + FILENAME_REFORMATED + ' /usr/share/.')
+
```

Comment:
Yeah, dunno why this is still here, this should be in master. but it's not. So it's here!